native/src/components/TtsContainer.tsx

```diff
@@ -110,6 +115,11 @@ const TtsContainer = ({ children }: TtsContainerProps): ReactElement => {
     async (index = sentenceIndex) => {
       const safeIndex = Math.max(0, index)
       const sentence = sentences[safeIndex]
+      if (!isLanguageSupported) {
+        close()
+        showSnackbar({ text: t('languageNotSupported') })
+        return
+      }
```

Comment:
🤔 I think this is the wrong place for this, in this case a `useEffect` with the only dependency `isLanguageSupported` feels like the better choice. At least it does not belong into the `play` function (the play function should not be called in the first place if the language is not supported).
🔧 I think showing a snackbar in this case is confusing to users as the context is missing and a general "your language is not supported" is not true in general. Therefore I would just silently close the player.

Reply:
To test this please:
- Switch to the English or German language (supported language).
- Open read-aloud function from kebab menu.
- Switch to unsupported language like Persian.
- here you can notice that the player is open and when you press play it will close() then show a snack bar in this case.

Reply:
I had to add visible and close to the dependency because eslint is complaining and visible is used here to prevent useEffect from calling `close()` at the first run and mess with how many times `stop()` has been called at `TtsContainer.spec.tsx`.

https://github.com/digitalfabrik/integreat-app/blob/fd7276a131844e92269d3a83758596600135ef6c/native/src/components/TtsContainer.tsx#L144-L148